eventhubs/client/azure-eventhubs/src/test/java/com/azure/eventhubs/CredentialInfoTest.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.eventhubs;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.experimental.theories.DataPoints;
+import org.junit.experimental.theories.Theories;
+import org.junit.experimental.theories.Theory;
+import org.junit.runner.RunWith;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Locale;
+
+@RunWith(Theories.class)
+public class CredentialInfoTest {
+    private static final String END_POINT_FORMAT = "sb://%s.%s";
```

Comment:
Since this endpoint format is a common string, we should add it as a package-private constant in ClientConstants. Then we don't have to redefine it all the time.

Reply:
Use public instead of package-private because the Test can't find it.

Reply:
Sounds good.